science/providers/python_build_standalone.py

```diff
@@ -155,6 +155,10 @@ def rank_compatibility(platform: Platform, target_triple: str) -> int | None:
                 match target_triple:
                     case "aarch64-unknown-linux-gnu":
                         return 0
+            case Platform.Linux_armv7l:
+                match target_triple:
+                    case "armv7-unknown-linux-gnueabihf":
+                        return 0
```

Comment:
I think adding another case for "armv7-unknown-linux-gnueabi" with rank 1 makes sense. Falling back to software floating point should be fine.